similarity/README.md

```diff
@@ -2,7 +2,47 @@
 
 This module specifies a simple API for NLP components that compare
 objects -- especially Strings -- and return a score indicating how
-similar they are.  It is used in our WordNet-, Named Entity-, embedding-, 
-and paraphrase-based similarity code to simplify integration of 
+similar they are.  It is used in our WordNet-, Named Entity-, embedding-,
+and paraphrase-based similarity code to simplify integration of
 different similarity resources.
- 
\ No newline at end of file
+
```

Comment:
Add a section at the end that covers LLM, and show how you run the LLM comparison that uses two different views. It's fine to summarize and point to the code, once you give the usage example.